db/migrations/README.md

```diff
@@ -0,0 +1,22 @@
+# Database Schema Changes
+
+## Philosophy
+* All database changes will be modeled in discrete schema migrations
+    * Code to roll back a migration will be included unless highly impracticable
+* Migrations will receive unique sequential integer identifiers.
+    * Engineers should keep aware of identifier collisions (and in some instances required migration order), and adjust their migration ID’s before merging with master, if necessary.
```

Comment:
If a migration is run on `0001` and a few days later `0002` is written, what happens if I run `up`?  Will both `0001` and `0002` be executed?  Or is it smart enough to know that only `0002` needs to be run?

Reply:
It's smart enough to know, by keeping track in a table named `database_migrations`.